run.py

```diff
@@ -847,9 +852,12 @@ def main():
             cwd, dirpath, outdir, suricata_config, args.verbose)
         try:
             results = test_runner.run()
-            passed += results["success"]
-            failed += results["failure"]
-            skipped += results["skipped"]
+            if results["failure"]:
```

Comment:
`results["failure"] > 0` would be better, wouldn't it ?

Reply:
Since there is no subtraction operation being performed on results and they are initialized with 0, it is guranteed to be positive.

Reply:
Yes, your code works.
This is a suggestion of code style.
The goal is to ensure that results will still be correct even if the above assumptions (like there is no subtraction) no longer are.
And it looks more readable/meaningful to me this way.

Reply:
Oh OK. Sorry for the misunderstanding. Sure, will do that.